### PR TITLE
fix(CreateEdit): Changed remaining characters text #293

### PR DESCRIPTION
**Pull request by wski**

### CODE REVIEW

File: app/js/components/createEdit/form/InputMixin.jsx [hunk 1]
```diff
@@ -8,18 +8,29 @@ var InputMixin = {
...
-        var CharLimit = (this.props.charLimit && this.state.value) ?
-            `${this.props.charLimit - this.state.value.length}/${this.props.charLimit} characters remaining`:
-            // if we do have a character limit, but value is undefined set it to the char limit max remaining
-            `${this.props.charLimit}/${this.props.charLimit} characters remaining`;
+        var CharLimit = (this.props.charLimit) ? `${this.props.charLimit}/${this.props.charLimit} characters remaining` : '';
+
+        if(this.props.charLimit && this.state.value){
+          if(this.props.charLimit - this.state.value.length < 0){
+            CharLimit = `${Math.abs(this.props.charLimit - this.state.value.length)} characters over the limit.`;
+          }else{
```
Comment (blaino) [inline app/js/components/createEdit/form/InputMixin.jsx]: Worth having a discussion some time about style. I think most of the style guides (e.g.  http://google.github.io/styleguide/javascriptguide.xml#Parentheses) say spaces before/after the else.

**Comment by blaino** [comment]
:+1:

**Comment by clarkjefcoat** [comment]
If you are one character under or over the message says "1 characters remaining" and "1 characters over the limit".

**Comment by wski** [comment]
@clarkjefcoat fixed it up a bit.

**Comment by clarkjefcoat** [comment]
:+1:

**Pull request merged**